package.json

```diff
@@ -11,7 +11,8 @@
     "geth": "./scripts/geth.sh",
     "dependency-graph": "lerna-dependency-graph -f pdf -o dependency-graph.pdf",
     "solc-bump": "node ./scripts/solc-bump.js",
-    "update": "lernaupdate"
+    "update": "lernaupdate",
+    "depcheck": "lerna exec --stream --no-bail dependency-check --ignore @truffle/contract-tests --ignore @truffle/dashboard-message-bus-e2e-test  -- --missing --verbose ."
```

Comment:
Oh, it's because they have no entry point, I see.  Um, so, wait, is this thing only checking the main code and not the test code?  (Still an improvement, but worth noting if that caveat is present.)

Reply:
I ignored them because they are both `private: true` and have no entry points. The tool has a [--entry](https://www.npmjs.com/package/dependency-check#--entry) flag if we want to include private packages at some later time.